### ISSUE TITLE
Setting the Cursor Blinking to solid changes the Cursor Style setting and changing the Cursor Style setting to block changes the Cursor Blinking setting.

### ISSUE THREAD

**Issue by carlod** [Microsoft/vscode#71680]
Issue Type: <b>Bug</b>

1. Set the Cursor Blinking to solid
2. Set the Cursor Style to block
3. Cursor Blinking changes the setting that was selected.
4. Go to Cursor Blinking and set it back to solid
5. Cusor Style changes the value that was selected.

VS Code version: Code 1.32.3 (a3db5be9b5c6ba46bb7555ec5d60178ecc2eaae4, 2019-03-14T23:43:35.476Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz (4 x 2496)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>surface_synchronization: enabled_on<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|15.92GB (9.07GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
debugger-for-chrome|msj|4.11.3
java|red|0.42.1
LiveServer|rit|5.5.1
vscode-java-debug|vsc|0.17.0

</details>

**Comment by alexandrudima** [comment]
@carlod I am sorry I don't fully understand the issue. I have configured the following settings:
```
    "editor.cursorBlinking": "solid",
    "editor.cursorStyle": "block",
```

This looks like this and appears to work fine AFAICT:
![image](https://user-images.githubusercontent.com/5047891/55543734-ef9d7600-56c9-11e9-95eb-704a42ee5b66.png)

Can you please explain in greater detail what is the issue?

**Comment by carlod** [comment]
Hi, the issue isn't with setting the Cursor Blinking and Cursor Style manually via the editing the settings,json file - it's setting them via the File ->Preferences ->Settings ->Text Editor-> Cursor menu. I've attached a video of the bug in action:

[VS Code Bug Solid Block Cursor.zip](https://github.com/Microsoft/vscode/files/3043959/VS.Code.Bug.Solid.Block.Cursor.zip)

**Comment by roblourens** [comment]
This should be fixed in 1.33
